tests_tf/test_mnist_tutorial_tf.py

```diff
@@ -0,0 +1,21 @@
+from inspect import getsourcefile
+import os
+import sys
+
+# Import the tutorial after evaluating its relative path
+current_dir = os.path.dirname(os.path.abspath(getsourcefile(lambda: 0)))
+current_dir = current_dir[:current_dir.rfind(os.path.sep)]
+tutorial_dir = os.path.join(current_dir, 'tutorials')
+sys.path.insert(0, tutorial_dir)
+import mnist_tutorial_tf
+sys.path.pop(0)
```

Comment:
if we keep the path modification code, it might be somewhat tricky to make it behave well if something fails. For example, if `import mnist_tutorial_tf` fails, maybe this results in `nosetests` being stuck in the wrong directory. This may be something we should handle with a `try` block and `finally` block. Even then, it could hard to tell whether or not to run the `pop` inside the `finally` block.